brands/xiaomi/README.md

```diff
@@ -14,23 +14,13 @@ Also, check out [Xiaomi-HyperOS-BootLoader-Bypass] by [MlgmXyysd](https://github
 
 ### China
 
-> 🧹 Missing a source for China Requirements.
-
-With the new policy, **You must have a Level 5 developer Xiaomi Community account.** </br>
-And here's a list of things you have to do to get that:
-- You must be a citizen of China.
-- You have to use HyperOS and report at least 1 bug per day. (????)
-- You have to submit at least 1 HyperOS suggestion per month. (????)
-- You must be an active Xiaomi Community user.
-
-Additional BS applies:
-- The permission to unlock is temporary and is granted for 1-year only.
-- You can only unlock 3 devices per-year.
-- You will not receive OTA updates.
-- your warranty will be void.
+[The bootloader CANNOT be unlocked on devices of the Mainland China region](https://gsmchina.com/xiaomi-will-no-longer-allow-bootloader-unlocking-5618/).
```

Comment:
I believe that the China thing started ~a month ago, so a more recent article would fit better. This one still talks about the old, short-lived requirements.